### PR TITLE
Update to match the new 1.18 height limit

### PR DESCRIPTION
**Pull request by CodeDoc537**
changed the max slider range in Anchor to match the new 128 blocks higher height limit in 1.18+

### CODE REVIEW

File: src/main/java/meteordevelopment/meteorclient/systems/modules/movement/Anchor.java [hunk 1]
```diff
@@ -25,7 +25,7 @@ public class Anchor extends Module {
         .name("max-height")
         .description("The maximum height Anchor will work at.")
         .defaultValue(10)
-        .range(0, 255)
+        .range(0, 383)
```
Comment (C10udburst) [inline src/main/java/meteordevelopment/meteorclient/systems/modules/movement/Anchor.java:28]: im pretty sure this is wrong.

1.18 coordinates are from `-64` to `320`

**Review by C10udburst — CHANGES_REQUESTED** [review]

File: src/main/java/meteordevelopment/meteorclient/systems/modules/movement/Anchor.java (hunk 1 above)
Comment (CodeDoc537) [reply src/main/java/meteordevelopment/meteorclient/systems/modules/movement/Anchor.java:28]: pre 1.18 was up to 256, so its the total height limit -1 so I just did the new height -1
320 + 64 = 384, 384 - 1 = 383

Comment (arlomcwalter) [reply src/main/java/meteordevelopment/meteorclient/systems/modules/movement/Anchor.java:28]: > im pretty sure this is wrong.
> 
> 
> 
> 1.18 coordinates are from `-64` to `320`

-64 to 320 and therefore its possible to be 383 (well more obviously) blocks above the bottom of the world

Comment (arlomcwalter) [reply src/main/java/meteordevelopment/meteorclient/systems/modules/movement/Anchor.java:28]: as in it is relative to the floor beneath you (iirc)

**Comment by soshimee** [comment]
It really depends on the server you are playing on though, since it can be changed.

File: src/main/java/meteordevelopment/meteorclient/systems/modules/movement/Anchor.java (hunk 1 above)
Comment (CodeDoc537) [reply src/main/java/meteordevelopment/meteorclient/systems/modules/movement/Anchor.java:28]: exactly

**Comment by CodeDoc537** [comment]
> It really depends on the server you are playing on though, since it can be changed.

well yeah, but its nice just to have the option

**Comment by offbeat-stuff** [comment]
```java
mc.world.getBottomY()
mc.world.getTopY()
```

**Comment by arlomcwalter** [comment]
its literally nothing to do with world height

**Comment by arlomcwalter** [comment]
> ```java
> 
> mc.world.getBottomY()
> 
> mc.world.getTopY()
> 
> ```
> 
> Can you use these two values

you can, but its nothing to do with this

**Comment by arlomcwalter** [comment]
> It really depends on the server you are playing on though, since it can be changed.

literally this is a relative value, there is no way you'd ever want to have it over like 20 but anyway its nothing to do with world height

**Pull request closed without merging**